tests/dataset/test_exclude.py

```diff
@@ -11,6 +11,8 @@
     ("data/.system-settings", ["data/.system"], False),
     ("my-data", ["data/.system"], False),
     ("my-data/.system", ["data/.system"], False),
+	("data/.system/cores", ["data/*/cores"], True),
+	("data/.system2/cores", ["data/*/cores"], False),
```

Comment:
Why won't be `data/.system2/cores` excluded by `data/*/cores`?

Reply:
It should, and the test does indeed fail. I think I meant to put `data/.system/cores2` and ran the test before saving that file.